examples/BUILD

```diff
@@ -45,16 +45,6 @@ cc_fuzz_test(
     dicts = ["dict_dir/valid.dict"],
 )
 
-# This target shows a fuzz test target with an invalid dictionary.
-# Building and runing empty_fuzz_test_with_invalid_dict should be ok,
-# but building and running empty_fuzz_test_with_invalid_dict_run are expected to get an error message
-# due to the invalid dictionary.
-cc_fuzz_test(
```

Comment:
Is this still being used or does it still make sense to have it around (e.g., for testing)? Mostly asking since the PR description sounded like this was mostly going to be a refactoring

Reply:
That's a great point - this was intended to be a refactoring, but it does indeed slightly change the semantics of the `cc_fuzz_test` rule: while the rule output stays the same, there are now more targets in the macro that are built by default. This test was assuming one of these targets (the dictionary) would not be built by default, so someone could manually check that this fuzz test would fail.

But since these manual "compilation failure" tests are rather brittle and less useful, I opted to remove this one for the time being.

I opened #95 to track the addition of a proper regression test.